spec.md

```diff
@@ -463,8 +463,7 @@ values include `None`, Booleans, numbers, and strings, and tuples
 composed from hashable values.  Most mutable values, such as lists,
```

Comment:
I noticed that the TOC that exists in Go spec has vanished from bazelbuild. Could you restore it?

Reply:
Done